src/Microsoft.ML.HalLearners/Microsoft.ML.HalLearners.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.HalLearners</IncludeInPackage>
```

Comment:
>HalLearners [](start = 35, length = 11)

Can't wait to train my HAL9000! ([https://pbs.twimg.com/profile_images/1788506913/HAL-MC2_400x400.png](https://pbs.twimg.com/profile_images/1788506913/HAL-MC2_400x400.png))

Reply:
In addition, you could also sign off on the PR :)

---
In reply to: [206715372](https://github.com/dotnet/machinelearning/pull/611#discussion_r206715372) [](ancestors = 206715372)